showcase/src/assets/translations/fr.json

```diff
@@ -393,6 +393,16 @@
         }
       }
     },
+    "FORMS": {
+      "NGX_FORMS": {
```

Comment:
Why is this block apart? Shouldn't it be in the the block of NGX_FORM_ERRORS? Moreover, I think it should be REACTIVE_FORM_ERRORS instead, so it is in sync with the actual page